src/log-manager/src/nginx/get_log_content.lua

```diff
@@ -62,23 +74,18 @@ end
 
 ngx.log(ngx.INFO, "get log from path"..log_path)
 
-if lfs.attributes(log_path, "mode") ~= "file" then
+if not util.is_path_under_log_folder(log_path) or not path.isfile(log_path) then
   ngx.log(ngx.ERR, log_path.." not exists")
   ngx.status = ngx.HTTP_NOT_FOUND
   return ngx.exit(ngx.HTTP_OK)
 end
 
 local logs
 if (tail_mode == "true") then
-  logs = io.popen("tail -c 16k "..log_path)
+  ngx.req.set_uri(log_path)
```

Comment:
https://www.openwall.com/lists/oss-security/2020/03/18/1

Reply:
We can not fix this issue. Just check the url to make sure it's validate